manifests/script-lab-prod.xml

```diff
@@ -17,7 +17,14 @@
     <AppDomain>https://github.com</AppDomain>
     <AppDomain>https://stackoverflow.com</AppDomain>
     <AppDomain>https://dev.office.com</AppDomain>
-    <AppDomain>https://script-lab-runner.azureedge.net</AppDomain>  
+    <AppDomain>https://localhost:3000</AppDomain>
```

Comment:
I think it makes it easier if we do have it everywhere (and esp. based on what you and I discussed, about being able to go back from production to localhost if you had originally sideloaded the dev one).  I don't see how it can be a security risk; the worst the user could do is hack their own selves.

But I'm also fine with removing it for prod.